tests/test_config.py

```diff
@@ -50,7 +50,7 @@ def config_none_db_url_values(base_config_values):
     """
     base_config_values["db_url"] = "None"
     yield base_config_values
-    os.environ.pop("DB_URL")
+    os.environ.pop("DB_URL", None)
```

Comment:
Why do we need to `pop` here instead of `get`

Reply:
We are removing the value from the current environment. This change just ensures it's a no-op if the variable is not defined.

Previously, the code was always setting DB_URL in the environment so the tests didn't notice.